redux/themeSlice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction, Slice } from "@reduxjs/toolkit";
+import { RootState } from "./rootReducer";
+
+export enum Theme {
+  DEFAULT = "theme-default",
+  DRACULA = "theme-dracula",
+}
+
+export type ThemeState = {
+  currentTheme: Theme;
+};
+
+const initialState: ThemeState = {
+  currentTheme: Theme.DRACULA,
```

Comment:
@vithushan19 - I think we should set the initial theme to light mode (DEFAULT) instead of dark (DRACULA). 

I noticed that there's a few components in dark mode that don't look good and need restyling:
- coaches page cards
- expandable containers 
- goals (text blends in with dark bg)